### PR TITLE
Update Simplified Chinese language translation

### PR DESCRIPTION
**Pull request by wcxu21**
Update Simplified Chinese language translation

### CODE REVIEW

File: RetroBar/Languages/中文(简体).xaml [hunk 1]
```diff
@@ -7,61 +7,76 @@
...
+    <s:String x:Key="show_multi_mon">在多个显示器上显示(_M)</s:String>
+    <s:String x:Key="show_quick_launch">显示快速启动(_Q)</s:String>
+    <s:String x:Key="select_location">选择位置(_S)...</s:String>
     <s:String x:Key="quick_launch_folder">快速启动 - 选择一个文件夹</s:String>
-    <s:String x:Key="show_window_previews">显示窗口预览 (缩略图) (_W)</s:String>
-    <s:String x:Key="use_software_rendering">使用软件渲染 (_U)</s:String>
-    <s:String x:Key="add_show_desktop_button">添加显示桌面按钮 (_D)</s:String>
-    <s:String x:Key="enable_auto_hide">自动隐藏任务栏 (_U)</s:String>
-    <s:String x:Key="lock_taskbar">锁定任务栏 (_L)</s:String>
+    <s:String x:Key="show_badges">Show _badges</s:String>
```
Comment (xoascf) [inline RetroBar/Languages/中文(简体).xaml:35]: Missing translation, Microsoft call the badges "锁屏提醒".

https://learn.microsoft.com/zh-cn/microsoft-edge/progressive-web-apps-chromium/how-to/notifications-badges
https://support.microsoft.com/zh-cn/windows/%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8-windows-11-%E4%B8%AD%E7%9A%84%E4%BB%BB%E5%8A%A1%E6%A0%8F-0657a50f-0cc7-dbfd-ae6b-05020b195b07

**Comment by wcxu21** [comment]
Thanks for your reminder. "Show badges" in the context of the taskbar usually refers to displaying notification badges or badges of applications, which appear on the taskbar icon to indicate unread messages, updates, or other notifications. In Chinese, this feature can be translated as "显示徽章" or "显示通知标记".

File: RetroBar/Languages/中文(简体).xaml (hunk 1 above)
Comment (wcxu21) [reply RetroBar/Languages/中文(简体).xaml]: Thanks for your reminder. "Show badges" in the context of the taskbar usually refers to displaying notification badges or badges of applications, which appear on the taskbar icon to indicate unread messages, updates, or other notifications. In Chinese, this feature can be translated as "显示徽章" or "显示通知标记".

Comment (wcxu21) [reply RetroBar/Languages/中文(简体).xaml]: The missing translation field has been updated, thanks for checking.

**Review by dremin — APPROVED** [review]
Thanks!

**Pull request merged**